src/index.html

```diff
@@ -459,21 +459,34 @@ <h1 slot="default">Wow</h1>
         
         <template id="app-button">
             <style>
-                button {
-                        background-color: var(--primary-background);
-                        color: white; 
-                        text-align: center;
-                        padding: 18px 25px;
-                        border-radius: 5px;
-                        font-weight: 500;
-                        margin-top: 50px;
-                        border-style: solid; 
-                        border-width: 1px;
-                        border-color: white;
-                    }
+                button{
+                    background-color: var(--primary-background);
+                    color: white; 
+                    text-align: center;
+                    padding: 16px 50px;
+                    border-width: 1px;
+                    margin-top: 50px;
+                    border-style: solid; 
+                    font-weight: 500;
+                    font-size: 12px;
+			
+                }
+                
+                .round{
```

Comment:
For this css selector `.round` you do not need the `background-color`,` color`, `text-align`, `padding`, `border-width`, `margin-top`,` border-style`, `font-weight`, `font-size`. Since all this properties are already on the button css selector they will still be there for the button component when you add the `.round class` to the button. You can test it out and see for yourself

Reply:
tested. I understand now. Thanks !